### ISSUE TITLE
facing error 

### ISSUE THREAD

**Issue by vedanthimte** [egandro/docker-qemu-arm#12]
![Screenshot_2023-01-07-12-59-04-586_com.termux.jpg](https://user-images.githubusercontent.com/112320570/211139467-867eaa5c-23fd-47a6-9ac3-9f496db6d02e.jpg)

**Comment by egandro** [comment]
Fix your connection. It's cristal clear where the error is :)

It can't download from alpinelinux.org...

**Comment by cmurtagh2010** [comment]
same error here. i can both ping and open in a browser the URL in question but the script hangs

**Comment by egandro** [comment]
Your device can't download from alpinelinux.org. The docker-qemu-arm script is running. That is inside alpine insidei the installation ... and that can't connect to alpine. You need to fix it.

**Comment by mkguru77** [comment]
I had the same issue and I have fixed it in my fork. I think it's dns issue.

**Comment by vip20** [comment]
@mkguru77 thank you for the solution. Seems a good fix. Why not create a PR for that. 

@deicpro your solution adds static ip, doesn't seem like a generic fix. Users will have to change that based on their routers.

**Comment by DeicPro** [comment]
> @mkguru77 thank you for the solution. Seems a good fix. Why not create a PR for that.
> 
> @DeicPro your solution adds static ip, doesn't seem like a generic fix. Users will have to change that based on their routers.

nope, it's userspace network.. has nothing to do with your router, so it's generic and works.

**Comment by vip20** [comment]
> > @mkguru77 thank you for the solution. Seems a good fix. Why not create a PR for that.
> > @DeicPro your solution adds static ip, doesn't seem like a generic fix. Users will have to change that based on their routers.
> 
> nope, it's userspace network.. has nothing to do with your router, so it's generic and works. You can set the IP as you like and it will not affect in nothing, only is important set as static because DHCP overwrite DNS with the fake virtual router IP which resolve nothing.
> 
> Please, don't talk and confirm about something if you didn't try it before.

I tried both and hence mentioned it here.

**Comment by PraveeshRamroop** [comment]
@DeicPro I tried your method and it works. But when I want to change my IP and gateway by editing the /etc/network/interfaces file, I can no longer ping 8.8.8.8. 

Why is that and how can I fix it?

**Comment by egandro** [comment]
I updated my script to the latest alpine version and fixed the DNS issues.

Please test!
